src/hive_mind_bridge/test/SocketToMoveByIntegrationTest.cpp

```diff
@@ -116,26 +116,24 @@ int main(int argc, char** argv) {
              "\tPayload: %d",
              statusReturnFunctionName.c_str(), arg0);
 
-    // Send moveBy request
-    serializer.serializeToStream(moveByMessageDTO);
-    std::this_thread::sleep_for(std::chrono::milliseconds(1000));
-
-    // Listen for a response
-    MessageDTO message;
-    deserializer.deserializeFromStream(message);
-    ResponseDTO response = std::get<ResponseDTO>(message.getMessage());
-    UserCallResponseDTO userCallResponse = std::get<UserCallResponseDTO>(response.getResponse());
-    FunctionCallResponseDTO functionCallResponse =
-        std::get<FunctionCallResponseDTO>(userCallResponse.getResponse());
-    GenericResponseDTO genericResponse = functionCallResponse.getResponse();
-    GenericResponseStatusDTO status = genericResponse.getStatus();
-    std::string details = genericResponse.getDetails();
-    ROS_INFO("RESPONSE FROM HOST (moveBy): \n"
-             "\tResponse status: %d\n"
-             "\tDetails: %s",
-             status, details.c_str());
-
-    // Listen for a return message
-    MessageDTO returnMessage;
-    deserializer.deserializeFromStream(returnMessage);
+    for (int i = 0; i < 5; i++) {
```

Comment:
why max 5? Maybe add a comment

Reply:
I wrote this. That's just for testing purposes. I could have made a while(true), but I wanted the code to end at some point.

Reply:
I dont recall writing those lines. I think it says that I'm the author because of the squash and merge of the fix/integration branch I did.